hw/ip/keymgr/doc/_index.md

```diff
@@ -119,53 +124,49 @@ Once the `OwnerRootKey` is created, software can request key manager to advance
 
 ### Invalid and Disabled State
 
-Note that `Invalid` and `Disabled` states are functionally equivalent.
-The main difference between the two is "how" the state was reached.
+`Invalid` and `Disabled` states are functionally very similar.
+The main difference between the two is "how" the states were reached and the entry behavior.
 
-`Disabled` state is reached through intentional software commands.
-While `Invalid` state is reached through life cycle disable or operational faults.
+`Disabled` state is reached through intentional software commands where the internal key, sideload key slots and software key are not wiped.
+While `Invalid` state is reached through life cycle deactivation or operational faults where the internal key, sideload key slots and software key are wiped.
 
 This also means that only `Invalid` is a terminal state.
-If after entering `Disabled` life cycle is disabled or a fault is encountered, the same [invalid entry procedure](#Invalid) is followed to bring the system to a terminal `Invalid` state.
+If after entering `Disabled` life cycle is deactivated or a fault is encountered, the same [invalid entry procedure](#Invalid) is followed to bring the system to a terminal `Invalid` state.
 
 If ever multiple conditions collide (a fault is detected at the same time software issues disable command), the `Invalid` entry path always takes precedence.
 
 ## Life Cycle Connection
-The function of the key manager is directly tied to the life cycle controller.
-During specific life cycle states, the key manager is explicitly invalidated.
+The function of the key manager is directly managed by the [life cycle controller]({{< relref "hw/ip/lc_ctrl/doc/#key-manager-en" >}}).
+
+Until the life cycle controller activates the key manager, the key manager does not accept any software commands.
+Once the key manager is activated by the life cycle controller, it is then allowed to transition to the various states previously [described](#key-manager-states).
 
-When invalidated, the following key manager behavior applies:
--  If the key manager has not been initialized, it cannot be initialized until life cycle enables key manager.
--  If the key manager has been initialized and is currently in a valid state (including `Disabled`), it immediately wipes its key contents with entropy (working state, sideload keys and software keys) and transitions to `Invalid`.
-   -  Note, unlike a normal software requested disable, this path does not gracefully interact with KMAC, instead the secret contents are forcibly wiped.
-   -  If there is an ongoing transaction with KMAC, the handshake with KMAC is still completed as usual, however the results are discarded and the value sent to KMAC are also not real.
--  Once the system settles to `Invalid` state, the behavior is consistent with `Disabled` state.
+When the life cycle controller de-activates the key manager, the key manager transitions to the `Invalid` state.
```

Comment:
"de-activates" -> "deactivates" to be consistent with how you wrote it everywhere else